README.md

```diff
@@ -1,13 +1,28 @@
 [![Build Status](https://travis-ci.com/bids-standard/bids-specification.svg?branch=master)](https://travis-ci.com/bids-standard/bids-specification)
 [![CircleCI](https://circleci.com/gh/bids-standard/bids-specification.svg?style=svg)](https://circleci.com/gh/bids-standard/bids-specification)
 
-![BIDS logo](BIDS_logo/BIDS_logo_black_transparent_background_crop.png)
 
-The Brain Imaging Data Structure (BIDS) is an emerging standard for the
+<img src="./BIDS_logo/BIDS_logo_black_transparent_background_crop.png" alt="bids-logo" width="600"/>
+
+The [Brain Imaging Data Structure (BIDS)](https://bids.neuroimaging.io) is an emerging standard for the
 organisation of neuroimaging data.
 
 In this repository, we develop the
 [BIDS specification](https://bids-specification.readthedocs.io/en/latest/).
 
-If you want to become a part of the BIDS community and contribute to the specification, please see our
-[contributing guidelines](./CONTRIBUTING.md).
+# Contributing to BIDS
+
+BIDS is a community-driven standard, and it depends on contributions from its members.
+We welcome new contributors, and we appreciate all contributions to the project!
+
+For a current list of our contributors, please see our [Contributors appendix](https://github.com/bids-standard/bids-specification/blob/master/src/99-appendices/01-contributors.md).
+
+When you're ready to get started, check out [our contributing guidelines](https://github.com/bids-standard/bids-specification/blob/master/CONTRIBUTING.md).
+
+Want to learn more about working with BIDS? Have a question, comment, or suggestion?
+Open or comment on one of our [NeuroStars issues](https://neurostars.org/tags/bids)!
```

Comment:
This feels a little out of place in the "Contributing section" and might be skipped over by a new user that doesn't consider themselves an aspiring contributor. Perhaps it would be better to place it as the lat thing before the sub-section?

Reply:
Moved and added mention of the Starter Kit based on a previous comment from @franklin-feingold